src/commands/project/deploy/cancel.ts

```diff
@@ -80,7 +80,7 @@ export default class DeployMetadataCancel extends SfCommand<DeployResultJson> {
       if (!this.jsonEnabled()) formatter.display();
       return formatter.getJson();
     } else {
-      const wait = flags.wait ?? Duration.minutes(deployOpts.wait);
+      const wait = flags.wait ?? Duration.minutes(deployOpts.wait ?? 33);
```

Comment:
Should we update the command help for cancel, report, and resume to note the 33 minute default?